test/errors.js

```diff
@@ -21,7 +21,6 @@ var REST_CODES = [
     { name: 'InvalidVersionError', code: 400 },
     { name: 'MissingParameterError', code: 409 },
     { name: 'NotAuthorizedError', code: 403 },
-    { name: 'PreconditionFailedError', code: 412 },
```

Comment:
These tests were added to ensure that changes in restify-error didn't break this repo. Since restify-errors is a breaking change, these tests had to be updated (arguably these tests should not belong in this repo, but don't recall the history here).